{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -43,7 +43,9 @@ paths = ["{{ cookiecutter.__project_slug }}", "vulture_allowlist.py"]
 
 [tool.poetry]
 name = "{{ cookiecutter.project_name }}"
+{%- if cookiecutter.__project_type!="agent_plugin" %}
 version = "v0.1.0"
+{%- endif %}
```

Comment:
I think we still want to default the version to v0.1.0, even for plugins?

Reply:
Well this version is for the project not the plugin. Our plugins project are not importable as packages (at least not now).